testing/jobs.pr.cloudbuild.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+
+- id: 'Lint'
+  name: 'golang:1.18-alpine'
+  dir: '${_SAMPLE_DIR}'
+  entrypoint: /bin/sh
+  args: 
+  - '-c'
+  - |
+    go install golang.org/x/tools/cmd/goimports@latest
+    if [ -n "$(goimports -d .)" ]; then
+      echo 'To fix this check, run "goimports -w ."'
+      exit 1
+    fi
+
+- id: 'Unit Tests'
+  name: 'golang:1.18-alpine'
+  dir: '${_SAMPLE_DIR}'
+  args: ['test', '-v']
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  dir: '${_SAMPLE_DIR}'
+  args: ['builds', 'submit', '--pack', 'image=gcr.io/${PROJECT_ID}/${_IMAGE}:${SHORT_SHA}'] # Tag docker image with git commit sha
```

Comment:
Since `latest` has some connotation for being literally the latest image, I wonder if this should build to something like `gcr.io/${PROJECT_ID}/previews/${_IMAGE}:${SHORT_SHA}` just to avoid ambiguity.

Reply:
The image name here is set to `jobs-pr` to avoid any accidents but maybe I shouldn't let that be a substitution?